test/specs/test.e2e.ts

```diff
@@ -15,24 +15,25 @@ describe('ODE basic workflow', () => {
 
         const simpleButton = await root.react$('SimpleButton', {props: {label: 'Get started'}})
         const focusTrap = await root.react$('FocusTrap')
-        const addButton = await $('div.MuiGrid-grid-md-4:nth-child(1)') // react$('AddButton') has problems when using toBeClickable. Using CSS selector instead.
+        const uploadYourDataButton = await $('.sidebar .MuiButton-outlined')
 
         await expect(focusTrap).toExist()
-        await expect(addButton).not.toBeClickable()
+        await expect(uploadYourDataButton).not.toBeClickable()
         simpleButton.click()
         await expect(focusTrap).not.toExist()
-        await expect(addButton).toBeClickable()
+        await expect(uploadYourDataButton).toBeClickable()
 
     }),
-    it('uploads a csv file, file navigator adds an node, and the file content gets display', async() => {
+    // TODO: re-enable when we find out why this is failing
+    it.skip('uploads a csv file, file navigator adds an node, and the file content gets display', async() => {
```

Comment:
@pdelboca @roll  skipping this temporarily until we find out what the problem is. we can still test this manually

Reply:
Sounds good! We will deal with this when keep exploring WebdriverIO.